src/main/resources/db/test/schema.sql

```diff
@@ -0,0 +1,869 @@
+-- MySQL dump 10.13  Distrib 5.7.10, for osx10.9 (x86_64)
```

Comment:
So does this static DB schema dump have to be manually kept up-to-date with migrations?  As we change the schema, do we have to regenerate this file every time?